.docker/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN chown -R www-data:www-data /var/www/.npm
 RUN mkdir -p /var/www/.composer
 RUN chown -R www-data:www-data /var/www/.composer
 
-RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
```

Comment:
I remember some issues with node 14 and back office assets, is everything ok on your side if you changed that?

Reply:
+1 for the concern, last time I checked there was a lot of errors with Node 16

Reply:
Yeah, I did double check that SF and BO open correctly with these changes.

Maybe others should try to do `docker compose up` with these changes to get more results?

Reply:
Also just checked and most (if not all) CI tasks already use node16 from actions/setup-node

Reply:
we had some issues with `make assets` on node 16, I'll check your changes there in a moment

Reply:
Just to resolve this conversation: All CI tasks already use node 16, the dev environment should also use node16. And I haven't seen any reports about problems with make assets in this PR.